src/contribute/design-documents.md

```diff
@@ -0,0 +1,87 @@
+- Static Display Data [[doc](https://docs.google.com/document/d/11enEB9JwVp6vO0uOYYTMYTGkr3TdNfELwWqoiUg5ZxM)]
+- Lateness (and Panes) in Apache Beam [[doc](https://s.apache.org/beam-lateness)]
+- Triggers in Apache Beam [[doc](https://s.apache.org/beam-triggers)]
+- Triggering is for sinks [[doc](https://s.apache.org/beam-sink-triggers)] (not implemented)
+- Pipeline Drain [[doc](https://docs.google.com/document/d/1NExwHlj-2q2WUGhSO4jTu8XGhDPmm3cllSN8IMmWci8)]
+- Pipelines Considered Harmful [[doc](https://s.apache.org/no-beam-pipeline)]
+
+### IO / Filesystem
+- IOChannelFactory Redesign [[doc](https://docs.google.com/document/d/11TdPyZ9_zmjokhNWM3Id-XJsVG3qel2lhdKTknmZ_7M)]
+- Configurable BeamFileSystem [[doc](https://docs.google.com/document/d/1-7vo9nLRsEEzDGnb562PuL4q9mUiq_ZVpCAiyyJw8p8)]
+- New API for writing files in Beam [[doc](http://s.apache.org/fileio-write)]
+
+### Metrics
+- Metrics API [[doc](http://s.apache.org/beam-metrics-api)]
+- I/O Metrics [[doc](https://s.apache.org/standard-io-metrics)]
+- Metrics extraction independent from runners / execution engines [[doc](https://s.apache.org/runner_independent_metrics_extraction)]
+
+### Runners
+- Runner Authoring Guide [[doc](https://s.apache.org/beam-runner-guide)] (obsoletes [[doc](http://s.apache.org/beam-runner-api)] and [[doc](https://s.apache.org/beam-runner-1-pager)])
+- Composite PInputs, POutputs, and the Runner API [[doc](https://s.apache.org/beam-runner-composites)]
+- Side Input Architecture for Apache Beam [[doc](https://s.apache.org/beam-side-inputs-1-pager)]
+
+### SQL / Schema
+- Streams and Tables [[doc](https://s.apache.org/beam-streams-tables)]
+- Streaming SQL [[doc](http://s.apache.org/streaming-sql-spec)]
+- Schema-Aware PCollections [[doc](https://docs.google.com/document/d/1tnG2DPHZYbsomvihIpXruUmQ12pHGK0QIvXS1FOTgRc)]
+
+### Portability
+- Fn API
+  - Apache Beam Fn API Overview [[doc](https://s.apache.org/beam-fn-api)]
+  - Processing a Bundle [[doc](https://s.apache.org/beam-fn-api-processing-a-bundle)]
+  - Progress [[doc](https://s.apache.org/beam-fn-api-progress-reporting)]
+  - Graphical view of progress [[doc](https://docs.google.com/document/d/1Dx18qBTvFWNqwLeecemOpKfleKzFyeV3Qwh71SHATvY)]
+  - Fn State API and Bundle Processing [[doc](https://s.apache.org/beam-fn-state-api-and-bundle-processing)]
+  - How to send and receive data [[doc](https://s.apache.org/beam-fn-api-send-and-receive-data)]
+  - Defining and adding SDK Metrics [[doc](https://s.apache.org/beam-fn-api-metrics)]
+  - SDK harness container contract [[doc](https://s.apache.org/beam-fn-api-container-contract)]
+  - Structure and Lifting of Combines [[doc](https://s.apache.org/beam-runner-api-combine-model)]
+- Cross-language Beam Pipelines [[doc](https://s.apache.org/beam-mixed-language-pipelines)]
+- SDK X with Runner Y using Runner API [[doc](https://s.apache.org/beam-job-api)]
+- Flink Portable Runner Overview [[doc](https://s.apache.org/portable-flink-runner-overview)]
+
```

Comment:
Launching portable pipeline on Flink Runner: https://docs.google.com/document/d/1xOaEEJrMmiSHprd-WiYABegfT129qqF-idUBINjxz8s

Reply:
Portability support table: https://docs.google.com/spreadsheets/d/1KDa_FGn1ShjomGd-UUDOhuh2q73de2tPz6BqHpzqvNI/edit#gid=0

Portability Prototype: https://s.apache.org/beam-portability-team-doc

Reply:
Thank you for other docs, done